Public/Sdk/SelfHost/BuildXL/StandaloneTest/StandaloneTestSupport.dsc

```diff
@@ -21,7 +21,7 @@ namespace StandaloneTestSupport {
             contents: [
                 StandaloneTestUtils.writeFile(a`main.dsc`, specContent),
                 StandaloneTestUtils.writeFile(a`module.config.dsc`, moduleContent),
-                Deployment.createFromDisk(toolDirectory.root, "*", true),
+                toolDirectory,

```

Comment:
globbing the root of a static directory is not safe because that folder is possibly being populated at build time